src/components/Sidebar/Tabs/DataTab.tsx

```diff
@@ -270,13 +302,14 @@ export default function DataTab({
         );
       }}
       position={i}
-      key={i}
+      key={`graph-pill-${i}`}
```

Comment:
Not necessary. Just use the index. This won't actually show up in the DOM so the string templating doesn't add anything.